src/datagen/generated/minecolonies/data/minecolonies/tags/items/reduceable_product_excluded.json

```diff
@@ -9,11 +9,13 @@
     "#forge:storage_blocks",
     "minecraft:glowstone",
     "minecraft:dried_kelp_block",
+    "minecraft:honey_block",
+    "minecraft:honeycomb_block",
```

Comment:
A couple of other storage blocks that should be excluded from reduction ... although that reminds me that I wanted to abstract this further.

Reply:
Now further abstracted.